cpp/test/CMakeLists.txt

```diff
@@ -316,6 +316,9 @@ if(BUILD_TESTS)
     test/neighbors/ann_cagra/test_float_uint32_t.cu
     test/neighbors/ann_cagra/test_int8_t_uint32_t.cu
     test/neighbors/ann_cagra/test_uint8_t_uint32_t.cu
+    test/neighbors/ann_cagra/test_float_uint64_t.cu
+    test/neighbors/ann_cagra/test_int8_t_uint64_t.cu
+    test/neighbors/ann_cagra/test_uint8_t_uint64_t.cu
```

Comment:
Adding these effectively doubles our [test binary size](https://downloads.rapids.ai/ci/raft/pull-request/1514/e790e82/cuda11_x86_64.tests_bench_prims.html):

File | Compile time | Size
-- | -- | --
test_float_uint32_t.cu.o | 43:55 min | 106.657 MB
test_float_uint64_t.cu.o | 42:24 min | 98.597 MB
test_uint8_t_uint32_t.cu.o | 41:31 min | 117.630 MB
test_int8_t_uint32_t.cu.o | 41:29 min | 116.698 MB
test_int8_t_uint64_t.cu.o | 40:04 min | 106.032 MB
test_uint8_t_uint64_t.cu.o | 39:56 min | 107.312 MB

See https://github.com/rapidsai/raft/pull/1499#issuecomment-1541583762 for potential impact on packages.

(Note that compile time is improved by #1428 by enabling parallel build for different template parameters, but binary size is not changed there.

Reply:
I have removed `test_int8_t_uint64_t` and `test_uint8_t_uint64_t`